### PR TITLE
Updates to latest wazero and wasmtime

### PR DESCRIPTION
**Pull request by codefromthecrypt**
This removes interrupt handling as it changed so much in implementation
that it needs to be recreated.

### CODE REVIEW

File: runner/wasm/wasmtime/wasmtime_go.go [hunk 1]
```diff
@@ -35,8 +35,7 @@ func newWasmInstance(store *wasmtime.Store, file string) *wasmtime.Instance {
 
 func newWasmStore() *wasmtime.Store {
 	config := wasmtime.NewConfig()
-	config.SetWasmModuleLinking(true)
```
Comment (codefromthecrypt) [inline runner/wasm/wasmtime/wasmtime_go.go:38]: removed https://github.com/bytecodealliance/wasmtime-go/pull/125

File: runner/wasm/wasmtime/x.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ func TransOutBytesReturn(store *wasmtime.Store, instance *wasmtime.Instance, cap
 	calloc := instance.GetFunc(store, "malloc")
 	aret, _ := calloc.Call(store, cap)
 
-	store.InterruptHandle()
+	store.SetEpochDeadline(1)
```
Comment (codefromthecrypt) [inline runner/wasm/wasmtime/x.go]: actually I'm not really sure what this should be doing.. the code changed a lot here https://github.com/bytecodealliance/wasmtime-go/pull/124/files#diff-9dc0dd17e62efefc839433c44c46c367888564b19b62fafcec3cd73410ba902bR52

File: runner/wasm/wasmtime/x.go [hunk 3]
```diff
@@ -27,16 +27,6 @@ func TransInBytesParam(store *wasmtime.Store, instance *wasmtime.Instance, in []
 	return aret, size, size
 }
 
-func TransOutBytesReturn(store *wasmtime.Store, instance *wasmtime.Instance, cap int32) (_, _, _ interface{}) {
```
Comment (codefromthecrypt) [inline runner/wasm/wasmtime/x.go:30]: this was dead code and also interrupt changed too much so it needs to be redone

https://github.com/bytecodealliance/wasmtime-go/pull/124

Comment (wuhuizuo) [reply runner/wasm/wasmtime/x.go:30]: I got the upgrade notify recently by `renovate` bot.
Should we keep with old wasmtime version, and waitting for another version?
I am  worring about wasmtime api stability.

@codefromthecrypt

Comment (codefromthecrypt) [reply runner/wasm/wasmtime/x.go:30]: wasmtime release policy is monthly so I think it best to just accept whatever happens.

**Pull request merged**